content/scripts/build.js

```diff
@@ -218,8 +218,9 @@ class Builder {
     this.logger = logger;
     this.selfHash = null;
     this.allTitles = {};
+    // TODO(peterbe): Evaluate why does this need to be its own data struct
+    // Can't it all just be part of this.allTitles??
```

Comment:
Followup!

I can no longer remember why `this.allPopularities` is its own thing when the `popularity` already gets included in the `this.allTitles`.